package.json

```diff
@@ -4,10 +4,8 @@
   "description": "A Social platform for the creative at heart",
   "main": "index.js",
   "scripts": {
-    "start": "node ./src/bin/www",
-    "devstart": "babel-watch ./src/index.js",
-    "build": "rm -rf dist && mkdir dist && babel src -s -d dist",
-   "test": " set NODE_ENV=test&& mocha -r @babel/register --recursive src/tests/ --exit"
+    "start": "node src",
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
Why did you remove the devstart and build script. I also notice you are not using the current version of our test script.

Reply:
I didn't remove them, in the version I checked out from, that's how it was. Let me just rebase it again

Reply:
If that was the code from the develop branch you checked out from and you didn't touch that code, why is it showing up as your change here. Ii this PR get's merged, what you currently have will override the current state of develop. I think the we all need to go over git, how we rebase with develop, recheck our workflow we have been following.

Reply:
Okay I see what you're saying. I've resolved it now, thanks.